selftests/unit/test_safeloader.py

```diff
@@ -47,6 +47,27 @@ def test_import_inside_class(self):
 
 class DocstringDirectives(unittest.TestCase):
 
+    NO_TAGS = [":AVOCADO: TAGS:FAST",
+               ":AVOCADO: TAGS=FAST",
+               ":avocado: mytags=fast",
+               ":avocado: tags",
+               ":avocado: tag",
+               ":avocado: tag=",
+               ":this is not avocado: tags=foo",
+               ":neither is this :avocado: tags:foo",
+               ":tags:foo,bar",
+               "tags=foo,bar"]
+
+    VALID_TAGS = {":avocado: tags=fast": set(["fast"]),
+                  ":avocado: tags=fast,network": set(["fast", "network"]),
+                  ":avocado: tags=fast,,network": set(["fast", "network"]),
+                  ":avocado: tags=slow,DISK": set(["slow", "DISK"]),
+                  ":avocado: tags=SLOW,disk,disk": set(["SLOW", "disk"]),
+                  ":avocado: tags=SLOW,disk, invalid": set(["SLOW", "disk"]),
+                  ":avocado: tags=SLOW,disk , invalid": set(["SLOW", "disk"]),
```

Comment:
OK, so the `invalid` is not there because of the tag `regexp`, right? I have to say I don't like this much. How about changing the regexp to allow spaces? Alternatively, if not already, could you please describe this in documentation with warning (or note) and ideally even here in the selftest and/or in the `get_docstring_directive_tags` function? Because from the function it looks as it should be there, but magically it's removed and I know I'll forget about the regexp and look for the reason later...

PS: Ideally I'd change the regexp to allow spaces, but visible note would also do...